spec/requests/order_items_spec.rb

```diff
@@ -13,4 +19,23 @@
       expect(JSON.parse(response.body)['data'].first['id']).to eq(order_item.id.to_s)
     end
   end
+
+  describe "#approval_requests" do
+    let!(:approval) { create(:approval_request, :order_item_id => order_item.id, :workflow_ref => "1") }
```

Comment:
@lindgrenj6 Missing :tenant_id

Reply:
We don't currently have tenants on approval_requests - is that something we'd want to add? the only way the user can access them is via `/order_items/{order_item_id}/approval_requests`, where order_items has tenancy enabled.

Reply:
@lindgrenj6 I think they should have tenancy on the approval requests.

Reply:
Added tenancy to `ApprovalRequest` model.